Samples/word-add-in-get-set-edit-openxml/README.md

```diff
@@ -63,19 +63,19 @@ The following screenshots (Figure 1a and Figure 1b) show how the document surfac
 
 Figure 1a. SampleDoc.docx document contents.
 
-![Screenshot of SampleDoc.docx document opened with images and formatted text.](description/CG_GetSetOOXML_fig01a.gif)
+![SampleDoc.docx document opened with images and formatted text.](description/CG_GetSetOOXML_fig01a.gif)
 
 Figure 1b. More SampleDoc.docx document contents.
 
-![Screenshot of SampleDoc.docx document opened with tables and SmartArt.](description/CG_GetSetOOXML_fig01b.gif)
+![SampleDoc.docx document opened with tables and SmartArt.](description/CG_GetSetOOXML_fig01b.gif)
 
 **Note**
 
 The sample uses Office Open XML (the OOXML coercion type) rather than HTML or plain text because OOXML coercion enables you to interact with virtually every type of content a user can insert in their document, such as in the examples you see in Figures 1a - 1b.
 
 Figure 2. Task pane appearance after using the **Get…** button to extract Office Open XML for selected content.
 
-![Screenshot of the add-ins task pane opened in Word.](description/add-in-screenshot.png)
```

Comment:
Style guidelines indicate we need to state the type of image.

Reply:
Thanks for bringing this up, David. I'll follow up with Linda on this as she mentioned that adding 'Screenshot of' or 'Image of' at the beginning of alt text is redundant since screen readers already announce image elements. She also mentioned that starting alt text with useful keywords improves SEO.